sdk/src/main/java/io/dapr/client/DaprGrpcClientAdapter.java

```diff
@@ -0,0 +1,155 @@
+package io.dapr.client;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.protobuf.*;
+import io.dapr.DaprGrpc;
+import io.dapr.DaprProtos;
+import io.dapr.utils.ObjectSerializer;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+
+public class DaprGrpcClientAdapter implements DaprClientAdapter {
```

Comment:
You can keep this named adapter.

No need to make it public, move it to package visibility. Customers should not instantiate this directly. We must provide a builder instead.